### PR TITLE
Fiche film

### PR DESCRIPTION
**Pull request by BenZen59**

### CODE REVIEW

**Review by Quentin-Macquart — COMMENTED** [review]
Hey!
I can't approve it just because of the different states of the actors. 
It's better to put just one state for [actors, setActors] and use a filter/map to render them. 
The rest is ok for me!

**Review by Quentin-Macquart — DISMISSED** [review]

File: src/components/FicheFilm/FicheFilm.jsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  const urlVideo = `https://www.youtube.com/embed/${trailer.key}`;
+
+  return (
+    <div>
+      <div className="titreVote">
+        <p>{title}</p>
+        <p className="vote">Users vote : {voteAverage}</p>
+      </div>
+      <div className="posterTrailer">
+        <img src={urlPoster} alt="" />
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.jsx]: Your alt is empty ! Not good for the SEO

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Your alt is empty !!

**Review by Hacene010 — APPROVED** [review]
That's better ;)

**Review by Quentin-Macquart — APPROVED** [review]

**Pull request merged**